ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 0.4.2
+* Fix package to be marked as typed
+* Fix `__repr__` to show something meaningful
+* Fix Ksuid comparisons
```

Comment:
want to pull in #13 and add a note about it before bumping this?

Reply:
No need, as it's not really user-visible. It's just there to make it typed (which is the first bullet point).